domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/AnyAssertions.kt

```diff
@@ -30,6 +30,9 @@ interface AnyAssertions {
 
     fun <T : Any?> toBeNull(subjectProvider: SubjectProvider<T>): Assertion
 
+    fun <T> isNotIn(subjectProvider: SubjectProvider<T>, expected: List<T>): Assertion
+
+
```

Comment:
This is part of the deprecated `Assertion` API. Please add your API functions to the new `Expect` api, which you can find in `apis/fluent-en_GB`. There, as @robstoll indicated in #180:

> * provide a `fun isNoneOf` with `signature (T, vararg T)` (see mapAssertions.kt -> contains as a guideline)
> * add `@since 0.9.0` to KDOC
> * provide a fun `isNotIn` with signature `Iterable<T>`
> * include a `require` that the `Iterable.iterator.hasNext()`
> * add `@since 0.9.0` to KDOC

Reply:
The function here is fine but please use `Expect` and not `SubjectProvider`. It was only introduce to ease the migration from Assert to Expect